lib/msal-react/docs/performance.md

```diff
@@ -0,0 +1,84 @@
+# Performance
+
+## How to configure `msal-react` to use your router's navigate function for client-side navigation
+
+By default, when msal.js needs to navigate from one page in your application to another it will reassign `window.location`, causing a full frame redirect to the other page and causing your application to re-render. If you're using a router this may be undesireable since many routers provide a method you can use to do "client-side" navigation and re-render only the parts of the page that need to be re-rendered.
+
+Currently there is one scenario where msal.js will navigate from one page in your application to another. If your application is **not** doing one or more of the following things you do not need to continue reading:
```

Comment:
```suggestion
Currently there is one scenario where msal.js will navigate from one page in your application to another. If your application **is** doing one or more of the following things, continue reading:
```
Don't love the double negative here (but maybe that's just me).

Reply:
Noted. I'll see if I can reword.